src/graphics/stk_scene_manager.cpp

```diff
@@ -833,7 +833,7 @@ PROFILER_POP_CPU_MARKER();
                 ShadowCmdBuffer = (DrawElementsIndirectCommand*)glMapBufferRange(GL_DRAW_INDIRECT_BUFFER, 0, 10000 * sizeof(DrawElementsIndirectCommand), GL_MAP_WRITE_BIT | GL_MAP_UNSYNCHRONIZED_BIT | GL_MAP_INVALIDATE_BUFFER_BIT);
             }
 
-            for (unsigned i = 0; i < 4; i++)
+            for (unsigned char i = 0; i < 4; i++)
```

Comment:
Doesn't make any difference, either it's in a register, or the loop is so slow (i.e. involving calls which force the register to be written back to memory) that it doesn't matter anyway.